### ISSUE TITLE
Add support for .erb files

### ISSUE THREAD

**Issue by Vuneu** [pranaygp/vscode-css-peek#6]
I can peek/go to definition on HTML files, but I do most of my development on Rails, any chance .erb support could be added?

Thanks!

**Comment by pranaygp** [comment]
Not super familiar with how the HTML language service works, but could you try just adding .erb as a valid file extension to your workspace/project settings.

The setting you're looking for is: `css_peek.searchFileExtensions`

**Comment by Vuneu** [comment]
I think you meant `css_peek.activeLanguages`, but yes! I added `"erb"` as an active language and now it's working.  

By the way, I noticed that if I define a CSS class like

```css
.card .content {
  /* */
} 
```

And I try to peek at content in a block like: 

```erb
<div class="card">
  <div class="front">
    <div class="cover">
    </div>
    <div class="user">
    </div>
    <div class="content text-center">
      <div class="main">
      </div>
    </div> 
  </div> 
</div>
```

I get the "no definition found for content" dialog.

**Comment by pranaygp** [comment]
Huh, that's a pretty big bug I overlooked when building this :/

I just realised the way I was finding the rule in CSS is by iterating through 'selectors' and comparing it to the class/id value.

Normally, this would be a comparison between '.content' and '.content', but in your case, it would compare '.content' with '.card .content' and fail :(

Anyway, I fixed it with a new commit. I'll release a patch soon to fix this. I'm also working on migrating over to using a CSS Language service instead which, if I'd used initially, wouldn't have caused this issue to begin with :P
